docs/serving/services/storage.md

```diff
@@ -1,12 +1,37 @@
 # Volume Support for Knative services
 
-By default Serving supports the mounting the [volume types](https://kubernetes.io/docs/concepts/storage/volumes): `emptyDir`, `secret`, `configMap` and `projected`. [PersistentVolumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) are supported but require a [feature flag](../configuration/feature-flags.md) to be enabled.
+You can provide data storage for Knative Services by configuring different volumes types.
+Serving supports mounting the [volume types](https://kubernetes.io/docs/concepts/storage/volumes): `emptyDir`, `secret`, `configMap` and `projected`.
+[PersistentVolumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) are supported but require a [feature flag](../configuration/feature-flags.md) to be enabled.
 
 !!! warning
     Mounting large volumes may add considerable overhead to the application's start up time.
 
 
-Here is an example of using a persistent volume claim with a Knative Service:
+Bellow there is an example of using a persistent volume claim with a Knative Service.
+
+## Prerequisites
+
+Before you can configure PVCs for a Service, this feature must be enabled in the `KnativeServing` ConfigMap:
```

Comment:
Sorry, this is still a bit confusing

> this feature must be enabled in the `KnativeServing` ConfigMap

It's either `KnativeServing` CR OR the config-map

Also then the hint

> Note that if you have installed Serving via the Knative operator then you need to set the above feature flags at the corresponding Serving CR.

Is unnecessary when the example is with the KnativeServing CR already.


Seems like in other pages we assume the YAML installation was used, so maybe it it easier to have a command here to patch the config-map, as we have [here](https://knative.dev/docs/install/yaml-install/serving/install-serving-with-yaml/#install-a-networking-layer)

> kubectl patch configmap/config-network \
  --namespace knative-serving \
  --type merge \
  --patch '{"data":{"ingress-class":"kourier.ingress.networking.knative.dev"}}'

Then we can also keep the note. Ideally we'll also have a section in the operator, describing the config-overwrite mechanism in general.

Reply:
> Is unnecessary when the example is with the KnativeServing CR already.

@ReToCode the yaml file you see is not with the Serving CR it is with the configmap I will use the patch just to clarify.